### PR TITLE
now empties /dev/tty before passing it to fzf

### PR DESCRIPTION
**Pull request by NightMachinary**
This seems to solve https://github.com/Aloxaf/fzf-tab/issues/206 .

### CODE REVIEW

**Comment by NightMachinary** [comment]
Update: Unfortunately, this only somewhat mitigates the problem. `fzf` still behaves weirdly for characters that are entered after `gdd` clears the pipe. I think it's ultimately an fzf bug, though I have not encountered this anywhere else except for `fzf-tab` ...

**Comment by NightMachinary** [comment]
@aloxaf Can you merge this? I recently changed my zsh plugin manager, and so my `fzf-tab` reverted to the latest master, without this patch. The bug appears a lot more often without this patch.

File: lib/-ftb-fzf [hunk 1]
```diff
@@ -64,6 +64,9 @@ fi
 
 cat > $tmp_dir/completions.$$
 
+
+_ftb_query="${_ftb_query}$(command gdd bs=1G count=1 status=none iflag=nonblock < /dev/tty 2>/dev/null)" || true
```
Comment (Aloxaf) [inline lib/-ftb-fzf:68]: I think this only works for macOS users, in Linux there is only `dd`.

You can use `$OSTYPE` to determine which one should be used here.

Comment (NightMachinary) [reply lib/-ftb-fzf]: @Aloxaf Ok, I added some checks which should be sufficient.

**Comment by Aloxaf** [comment]
LGTM

**Comment by NightMachinary** [comment]
Thanks! <3

**Pull request merged**